src/expr/src/scalar.proto

```diff
@@ -612,7 +612,6 @@ message ProtoBinaryFunc {
         google.protobuf.Empty mz_acl_item_contains_privilege = 184;
         google.protobuf.Empty age_timestamp = 185;
         google.protobuf.Empty age_timestamp_tz = 186;
-        google.protobuf.Empty age_current = 187;
```

Comment:
This needs to be marked as reserved in the `.proto` file https://protobuf.dev/programming-guides/proto3/#fieldreserved, including the name

Reply:
Looks like it was added in the first commit and removed in the second, so this should be fine.

Btw, protobuf compat issues should now be caught by CI as well!

Reply:
I didn't know either of these things :D ty

Reply:
Exactly, we decided to remove `age(ts)` for now, which is `age_current`, because it would make more sense to implement SQL features like `current_date` and then use those in the implementation